### ISSUE TITLE
Alignment and Pretoucher issue on M1 mac 

### ISSUE THREAD

**Issue by RobAustin** [OpenHFT/Chronicle-Queue#837]
test have been ignored on M1 Mac, due to possible Alignment issue.

To access the M1 mac in the TC cluster - see https://docs.google.com/document/d/1-nsInZ9X4YDDrl_3te5RhQRgd-f6e68dVWC2h804M-s/edit

effecting the following tests : ( see https://teamcity.chronicle.software/buildConfiguration/OpenHFT_ChronicleQueue4_SnapshotMac/566375 ) 

```
StuckQueueTest.testnet.openhft.chronicle.queue.impl.single
PretoucherStateTest.pretouchnet.openhft.chronicle.queue.impl.single
PretoucherStateTest.pretouchLongBreaknet.openhft.chronicle.queue.impl.single
PretoucherStateTest.pretouchLeapnet.openhft.chronicle.queue.impl.single
```

for example:
```
StuckQueueTest.testnet.openhft.chronicle.queue.impl.single

Stacktrace
Copy to clipboard
java.lang.AssertionError: mis-aligned
  at net.openhft.chronicle.queue.impl.single.StuckQueueTest.test(StuckQueueTest.java:63)
------- Stdout: -------
--- !!meta-data #binary
header: !SCQStore {
wireType: !WireType BINARY_LIGHT,
writePosition: [
0,
0
],
roll: !SCQSRoll {
length: !int 60000,
format: yyyyMMdd-HHmm,
epoch: 0
},
indexing: !SCQSIndexing {
indexCount: !short 2048,
indexSpacing: 16,
index2Index: 442,
lastIndex: 0
```

**Comment by RobAustin** [comment]
also this test ChronicleQueueIndexTest.checkTheEOFisWrittenToPreQueueFileWritingDocumentMetadata  - see https://teamcity.chronicle.software/viewLog.html?buildId=566375&tab=buildResultsDiv&buildTypeId=OpenHFT_ChronicleQueue4_SnapshotMac

```
ChronicleQueueIndexTest.checkTheEOFisWrittenToPreQueueFileWritingDocumentMetadata
--
First failure: #4104  Changes (44) 20 Apr 21 12:09java.lang.AssertionError: mis-aligned 	at net.openhft.chronicle.queue.ChronicleQueueIndexTest.checkTheEOFisWrittenToPreQueueFileInner(ChronicleQueueIndexTest.java:86) 	at net.openhft.chronicle.queue.ChronicleQueueIndexTest.checkTheEOFisWrittenToPreQueueFileWritingDocumentMetadata(Chro```

**Comment by JerryShea** [comment]
Is this OpenHFT/Chronicle-Bytes#361 a similar problem (Intel Mac)

**Comment by JerryShea** [comment]
@minborg assume you have not started this one? If not, Vachagan will take!

**Comment by minborg** [comment]
Great!

**Comment by RobAustin** [comment]
I'll take a look at this.

**Comment by RobAustin** [comment]
this test runs ok on the mac 

net.openhft.chronicle.queue.ChronicleQueueIndexTest#checkTheEOFisWrittenToPreQueueFileWritingDocumentMetadata


![image](https://user-images.githubusercontent.com/6506056/170019766-47d05c07-b5dd-4cf3-93cc-c7aa0c72413e.png)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.23ea8](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.23ea8), [BOM-2.23ea68](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23ea68)
